aws/load-balancer.tf

```diff
@@ -43,6 +43,26 @@ resource "aws_lb_listener" "rancher_https" {
   }
 }
 
+resource "aws_s3_bucket" "rancher_lb_access_logs" {
+  bucket        = "wandera-${var.account_name}-rancher-access-logs"
```

Comment:
Please make this optional using a count on var length so we can switch on and off from the top level.

Reply:
This should not have the word Wandera in it as this is OSS. Please allow passing bucket name in from top level.